apps/client/index.html

```diff
@@ -1,31 +1,27 @@
 <!doctype html>
-<html lang='en'>
-<head>
-  <meta charset='utf-8' />
-  <base href="/">
-  <link rel='icon' href='/favicon.ico' />
-  <meta name='viewport' content='width=device-width, initial-scale=1' />
-  <meta name='theme-color' content='#101010' />
-  <meta name='ontime' content='ontime - time keeping for live events' />
-  <link rel='apple-touch-icon' href='/ontime-logo.png' />
-  <link
-    rel='icon'
-    type='image/png'
-    href='ontime-logo.png'
-  />
-  <link rel='manifest' href='/site.webmanifest' />
-  <link rel='manifest' href='/manifest.json' />
-  <title>ontime</title>
-  <style>
-    body,
-    html {
-      background-color: rgba(0, 0, 0, 0) !important;
-    }
-  </style>
-</head>
-<body>
-<noscript>You need to enable JavaScript to run this app.</noscript>
-<div id='root'></div>
-<script type='module' src='/src/index.tsx'></script>
-</body>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <base href="/" />
+    <link rel="icon" href="/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#101010" />
+    <meta name="ontime" content="ontime - time keeping for live events" />
+    <link rel="apple-touch-icon" href="/ontime-logo.png" />
+    <link rel="icon" type="image/png" href="/ontime-logo.png" />
+    <link rel="manifest" href="/site.webmanifest" />
+    <link rel="manifest" href="/manifest.json" />
+    <title>ontime</title>
+    <style>
+      body,
+      html {
+        background-color: #101010 !important;
```

Comment:
I think there was a reason this was all black, but I don't remember
maybe leave it for now?
 Unless you know

Reply:
good call, I seem to remember this was to avoid colour flashes on load, but I may be remembering wrong, anyway, it is not worth changing